packages/ui/src/components/sidebar/Sidebar.tsx

```diff
@@ -91,6 +92,13 @@ export function Sidebar({setOpen}: SidebarProps) {
         {{
           type: TabType.Space,
         }}
+        {{
```

Comment:
I'd move it up so that it's right under the project selection icon.
This way all links are grouped in one place instead of being scattered.